classes/core/Core.inc.php

```diff
@@ -42,6 +42,24 @@ static function getBaseDir() {
 		return $baseDir;
 	}
 
+	/**
+	 * Get the path in which file caches will be stored.
+	 * @return string The full path to the file cache directory
+	 */
+	static function getFileCachePath() {
+		return Core::getBaseDir() . DIRECTORY_SEPARATOR . 'cache';
+	}
+
+	/**
+	 * Get a stash pool instance to use for file-based caching.
+	 * @return Stash\Driver\FileSystem A stash pool instance.
```

Comment:
Is this the pool or the driver? I'm a little confused on the terminology, but I think it's `new Stash\Pool()` that returns the pool, right?

Reply:
It's the driver, not the pool -- I'll fix the self-doc.